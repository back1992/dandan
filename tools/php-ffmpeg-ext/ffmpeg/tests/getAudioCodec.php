<?php
$mov = new ffmpeg_movie(dirname(__FILE__) . '/test_media/robot.avi');
printf("ffmpeg getAudioCodec(): %s\n", $mov->getAudioCodec());
?>
