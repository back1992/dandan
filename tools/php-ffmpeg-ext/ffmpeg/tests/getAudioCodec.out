Fatal error: Class 'ffmpeg_movie' not found in /var/www/html/dandan/tools/php-ffmpeg-ext/ffmpeg/tests/getAudioCodec.php on line 2